extensions/dacpac/src/wizard/pages/deployConfigPage.ts

```diff
@@ -183,6 +190,10 @@ export class DeployConfigPage extends DacFxConfigPage {
 			return false;
 		}
 		let values = await this.getDatabaseValues();
+		// If there are no databases, call the disableUpgradeRadioButton function.
+		if (values[0] === undefined) {
```

Comment:
there can be multiple servers in the dropdown. If the first one has no dbs and the upgrade existing database radio button is disabled, does it get enabled again if the target server is changed to one that does have databases?

Reply:
From what I've understood from the code, I believe it should be changed accordingly, as the check is done using the getDatabaseValues() function, and this should fetch the databases from its respective servers in the dropdown and based on that the radio button DOM should be changed accordingly.

Reply:
I left a comment to add an else to enable this.upgradeExistingRadioButton if there are databases, just in case